### PR TITLE
support multiple teams

### PR DESCRIPTION
**Pull request by AriGute**

### CODE REVIEW

**Comment by AriGute** [comment]
image

File: src/components/TeamGeneratorPage/PlayerCard.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 
-const playerCard = ({ player, onRemovePlayer }) => {
+const playerCard = ({ player, onRemovePlayer, team }) => {
 	const drag = (e) => {
```
Comment (Ronielo) [inline src/components/TeamGeneratorPage/PlayerCard.js:4]: this is the eventListener?

File: src/components/TeamGeneratorPage/services/teamsHandler/TeamsHandler.js
Comment (Ronielo) [inline src/components/TeamGeneratorPage/services/teamsHandler/TeamsHandler.js:1]: you basically implemented a MobX on your project without knowing you are doing it

File: src/components/TeamGeneratorPage/TeamsDisplay/TeamsDisplay.js [hunk 2]
```diff
@@ -37,8 +32,17 @@ const TeamsDisplay = ({ playerList }) => {
...
-		<div className='flex items-center rounded'>
-			<TeamCompCard list={teamA} name={teams.A}></TeamCompCard>
+		<div className=' flex flex-col items-center my-4'>
+			<AddRemoveTeam setPreTeams={setPreTeams}></AddRemoveTeam>
+			<div className='flex items-center justify-center flex-wrap'>
+				{preTeams.length === 0 ? (
+					<p className=' text-red-500 m-5'>There is 0 teams</p>
+				) : (
+					preTeams.map((preTeam, i) => {
+						return <TeamCompCard teamSet={preTeam} name={i} key={i}></TeamCompCard>;
```
Comment (Ronielo) [inline src/components/TeamGeneratorPage/TeamsDisplay/TeamsDisplay.js:42]: remove </TeamCompCard>

Comment (Ronielo) [inline src/components/TeamGeneratorPage/TeamsDisplay/TeamsDisplay.js]: remove the closing tag

File: src/components/TeamGeneratorPage/common/TeamCompCard.js [hunk 3]
```diff
@@ -1,34 +1,47 @@
...
 				<ul>
-					{list.map((item, i) => {
+					{teamList.map((item, i) => {
 						return (
-							<PlayerCard key={item + i} player={item} onRemovePlayer={onRemovePlayer}></PlayerCard>
+							<PlayerCard
+								key={item + i}
+								player={item}
+								onRemovePlayer={onRemovePlayer}
+								team={teamName}></PlayerCard>
```
Comment (Ronielo) [inline src/components/TeamGeneratorPage/common/TeamCompCard.js:44]: remove closing tag

File: src/components/TeamGeneratorPage/services/teamsHandler/TeamsHandlerStorage.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import TeamsHandler from './TeamsHandler';
+
+export default class TeamsHandlerStorage {
+	static get() {
+		const data = localStorage.getItem('player_list');
```
Comment (Ronielo) [inline src/components/TeamGeneratorPage/services/teamsHandler/TeamsHandlerStorage.js:5]: why are you using the localStorage?

**Review by Ronielo — APPROVED** [review]
overall its fine, left some comments to get into context and a few eye catching things

File: src/components/TeamGeneratorPage/PlayerCard.js (hunk 1 above)
Comment (AriGute) [reply src/components/TeamGeneratorPage/PlayerCard.js:4]: `drag` is a callback function for the `onDragStart` event

File: src/components/TeamGeneratorPage/services/teamsHandler/TeamsHandlerStorage.js (hunk 4 above)
Comment (AriGute) [reply src/components/TeamGeneratorPage/services/teamsHandler/TeamsHandlerStorage.js:5]: to store players names and pre defined teams

**Pull request merged**